openwisp_monitoring/monitoring/configuration.py

```diff
@@ -0,0 +1,512 @@
+        'charts': {
+            'uptime': {
+                'type': 'bar',
+                'title': _('Uptime'),
+                'description': _(
+                    'A value of 100% means reachable, 0% means unreachable, values in '
+                    'between 0% and 100% indicate the average reachability in the '
+                    'period observed. Obtained with the fping linux program.'
+                ),
+                'summary_labels': [_('Average uptime')],
+                'unit': '%',
+                'order': 200,
+                'colorscale': {
+                    'max': 100,
+                    'min': 0,
+                    'label': _('Reachable'),
+                    'scale': [
+                        [0, '#c13000'],
+                        # [0.33, '#ef7d2d'],
+                        [0.5, '#deed0e'],
+                        [1, '#7db201'],
+                    ],
+                    'map': [
+                        [100, '#7db201', _('Reachable')],
+                        [33, '#deed0e', _('Partly reachable')],
+                        [None, '#c13000', _('Unreachable')],
+                    ],
+                    'fixed_value': 100,
+                },
+                'query': chart_query['uptime'],
+            },
+            'packet_loss': {
+                'type': 'bar',
+                'title': _('Packet loss'),
+                'description': _(
+                    'Indicates the percentage of lost packets observed in ICMP probes. '
+                    'Obtained with the fping linux program.'
+                ),
+                'summary_labels': [_('Average packet loss')],
+                'unit': '%',
+                'colors': [DEFAULT_COLORS[3]],
+                'order': 210,
+                'query': chart_query['packet_loss'],
+            },
+            'rtt': {
+                'type': 'scatter',
+                'title': _('Round Trip Time'),
+                'description': _(
+                    'Round trip time observed in ICMP probes, measuered in milliseconds.'
+                ),
+                'summary_labels': [
+                    _('Average RTT'),
+                    _('Average Max RTT'),
+                    _('Average Min RTT'),
+                ],
+                'unit': f' {_("ms")}',
+                'order': 220,
+                'query': chart_query['rtt'],
+            },
+        },
+        'alert_settings': {'operator': '<', 'threshold': 1, 'tolerance': 0},
+        'notification': {
+            'problem': {
+                'name': 'Ping PROBLEM',
+                'verb': 'cannot be reached anymore',
+                'level': 'warning',
+                'email_subject': _(
+                    '[{site.name}] {notification.target} is not reachable'
+                ),
+                'message': _(
+                    'The device [{notification.target}] {notification.verb} anymore by our ping '
```

Comment:
Watch out, the markdown link has been removed and only the square brackets have remained:

![Screenshot from 2020-08-07 21-11-40](https://user-images.githubusercontent.com/841044/89700362-2d7c6100-d8f3-11ea-94e8-a41d1c6ee62f.png)

The device should be linked as the word "default" in the other sample notifications.
Applies to the other alerts as well.

If in doubt, ask @pandafy.

Reply:
This has been fixed now.